cmd/snap-bootstrap/cmd_initramfs_mounts_test.go

```diff
@@ -310,7 +310,7 @@ func (s *initramfsMountsSuite) TestInitramfsMountsRunModeStep2(c *C) {
 `, s.runMnt))
 }
 
-func (s *initramfsMountsSuite) TestInitramfsMountsRunModeBaseSnapUpgradeHappy(c *C) {
```

Comment:
the diff that the merge commit created for this file is a bit confusing to read, I'd suggest reading the tests in full rather than looking at the diff